types.json

```diff
@@ -42,5 +42,9 @@
     {
         "code": "treachery",
         "name": "Treachery"
+    },
+    {
```

Comment:
This has already been added by the TSK merge, so you can remove this. (I don't think it causes a problem if it's re-added either...)